docs/source/deepinv.physics.rst

```diff
@@ -1,111 +1,380 @@
 .. _physics:
 
 Physics
-=======
+=========
 
-This package contains a large collection of forward operators appearing in imaging applications.
+Introduction
+------------
 
-The operators and are of the form
+This package contains a large collection of forward operators appearing in imaging applications.
+The acquisition models are of the form
 
 .. math::
 
-    y = \noise{\forw{x}}
+    y = \noise{\forw{x, \theta}}
+
+where:
+
+* :math:`x\in\xset` is an image
+* :math:`y\in\yset` are the measurements
+* :math:`\theta` is an optional parameter describing the acquisition system (e.g. point spread function in optics, sampling locations in tomography or MRI)
+* :math:`A:\xset\mapsto \yset` is a deterministic (linear or non-linear) operator capturing the physics of the acquisition
+* :math:`N:\yset\mapsto \yset` is a mapping which characterizes the noise affecting the measurements.
+
+This makes it possible to consider various problems including:  
 
-where :math:`x\in\xset` is an image of :math:`n` pixels, :math:`y\in\yset` are the measurements of size :math:`m`,
-:math:`A:\xset\mapsto \yset` is a deterministic (linear or non-linear) mapping capturing the physics of the acquisition
-and :math:`N:\yset\mapsto \yset` is a stochastic mapping which characterizes the noise affecting the measurements.
+* Regular inverse problems: recovering :math:`x` from :math:`y` and :math:`A` 
+* Calibration problems: recovering :math:`\theta` from :math:`x` and :math:`y`
+* Blind inverse problems: recovering :math:`x` and :math:`\theta` from :math:`y`
+* Computational imaging: optimizing :math:`\theta` and the parameters of a reconstruction mapping jointly
 
 
-All forward operators inherit the structure of the ``Physics`` class.
+Operators: general concepts
+----------------------------
+
+All forward operators inherit the structure of the :class:`Physics` class.
 
 .. autosummary::
-   :toctree: stubs
    :template: myclass_template.rst
    :nosignatures:
 
    deepinv.physics.Physics
 
+They are :class:`torch.nn.Module` which can be called with the ``forward`` method. 
+If the operator :math:`A` is parameterized as :math:`A(\theta)`, 
+the ``forward`` method can be called with a dictionary of parameters as an extra input.
+The following example shows how operators and their parameter can be instantiated and called
+
+.. doctest::
+
+   >>> import torch
+   >>> import deepinv as dinv
+   >>>
+   >>> from deepinv.physics import Blur
+   >>> x = torch.rand((1, 1, 16, 16))
+   >>> theta = torch.ones((1, 1, 2, 2)) / 4 # a basic 2x2 averaging filter
+   >>> # %% A first possibility
+   >>> physics = Blur(filter=theta) # we instantiate a blur operator with its convolution filter
+   >>> y = physics(x)
```

Comment:
```suggestion
   >>> y = physics.A(x)
```

I think it is important to make a distinction between this section where we explain how to apply an operator (that is `A`, i.e. `physics.A(...)`, which is different from `physics(...)` that contains that noise.